libphobos/m4/druntime/libraries.m4

```diff
@@ -16,6 +40,17 @@ AC_ARG_ENABLE(thread-lib,
 ])
 
 
+# DRUNTIME_LIBRARIES_TIME
+# -----------------------
++# Check whether -lrt is required for sched_yield, nanosleep, or clock_gettime.
+AC_DEFUN([DRUNTIME_LIBRARIES_TIME],
+[
+AC_SEARCH_LIBS([sched_yield], [rt])
+AC_SEARCH_LIBS([nanosleep], [rt])
+AC_SEARCH_LIBS([clock_gettime], [rt])
+])
+
```

Comment:
Only objects files, though but so long as all pragma declarations in all imports are passed to the backend...